app/src/org/commcare/connect/ConnectManager.java

```diff
@@ -985,4 +986,11 @@ public static String generatePassword() {
 
         return password.toString();
     }
+
+    public static boolean shouldShowJobStatus(Activity activity) {
```

Comment:
StandardHomeActivityUIController.getHiddenButtons an call this method to determine whether to add "connect" to the list of hidden buttons.